src/react-components/room/LoadingScreen.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { LoadingScreen } from "./LoadingScreen";
+import logoSrc from "../../assets/images/company-logo.png";
+
+export default {
+  title: "LoadingScreen"
+};
+
+export const Tip = () => (
+  <LoadingScreen
+    logoSrc={logoSrc}
+    message="Loading objects 2/14"
+    bottomHeader="Tip:"
+    bottomMessage="Press the Q & E keys to turn left and right."
+  />
+);
+
+Tip.parameters = {
+  layout: "fullscreen"
+};
+
+export const WhatsNew = () => (
+  <LoadingScreen
+    logoSrc={logoSrc}
+    message="Loading objects 2/14"
+    bottomHeader="What's New?"
+    bottomMessage={
+      <>
+        You can now set the default locale in your preferences.{" "}
+        <a href="#" target="_blank">
+          Read More
+        </a>
+      </>
+    }
+  />
+);
```

Comment:
These are cool pages. Would love for the tip/what's new to change every few seconds, or fade in after a second or two of loading. (It's best for loading to be fast enough that you don't need it, but that's not where we are now for a lot of scenes. However when you refresh and have everything cached it should be really fast -- might be better for there not to be a tip for the first second or two so that you don't get halfway thru reading it and then have it disappear on you).